src/Castle.Core.Tests/TestDictionaryValidator.cs

```diff
@@ -47,11 +47,11 @@ public bool IsValid(IDictionaryAdapter dictionaryAdapter)
 		public string Validate(IDictionaryAdapter dictionaryAdapter)
 		{
 			List<String> errors = new List<string>();
-			var globalRules = AttributesUtil.GetTypeAttributes<IValidationRule>(dictionaryAdapter.Meta.Type);
+			var globalRules = AttributesUtil.GetTypeAttributes<ValidationRuleAttribute>(dictionaryAdapter.Meta.Type);

```

Comment:
I can't think of a reason it would be needed, and since it is in the unit tests (not the library) I don't think there is any point spending more time trying to discover if it is.